### PR TITLE
PR 03 - Coveralls

### PR DESCRIPTION
**Pull request by ghost**
[Link to PT chore](https://www.pivotaltracker.com/story/show/167314825)
[Documentation used for install](https://class.craftacademy.co/courses/course-v1:CraftAcademy+CA-CC-02+2019/courseware/cb5cc85a2e4a416a8d323d355fbcc799/fba6d0747a3d47a5b7cc4f802f8a997a/1?activate_block_id=block-v1%3ACraftAcademy%2BCA-CC-02%2B2019%2Btype%40vertical%2Bblock%400cacbf6b115a40b180258616d35823a7)

1. Installed coveralls gem
2. Configured spec files to use coveralls
3. Created new rake task for coveralls

### CODE REVIEW

File: coverage/.resultset.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (oliverochman) [inline coverage/.resultset.json]: you need to add the coverage folder to the `.gitignore`

File: features/support/env.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 require 'cucumber/rails'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```
Comment (oliverochman) [inline features/support/env.rb:2]: these two lines need to be on top of the file

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 require 'spec_helper'
+require 'coveralls'
+Coveralls.wear_merged!('rails')
```
Comment (oliverochman) [inline spec/rails_helper.rb:2]: these two lines need to be on top of the file

File: coverage/.resultset.json (hunk 1 above)
Comment (oliverochman) [reply coverage/.resultset.json]: http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: coverage/.resultset.json (hunk 1 above)
Comment (ghost) [reply coverage/.resultset.json]: Added to .gitignore and successfully removed folder from repository.

File: features/support/env.rb (hunk 2 above)
Comment (ghost) [reply features/support/env.rb:2]: Done.

File: spec/rails_helper.rb (hunk 3 above)
Comment (ghost) [reply spec/rails_helper.rb:2]: Done.

**Review by oliverochman — APPROVED** [review]

**Pull request merged**